tests/Unit/Subscriber/Core/MappingSubscriberTest.php

```diff
@@ -43,11 +44,16 @@ public function setUp()
         $this->persistEvent->getLocale()->willReturn('de');
         $this->persistEvent->getAccessor()->willReturn($this->accessor);
 
+        $this->manager = $this->prophesize(DocumentManagerInterface::class);
```

Comment:
Not needed, if the unused call of the `getManager` method of the `MappingSubscriber` is removed.